avalon/tools/cbloader/model.py

```diff
@@ -72,16 +72,23 @@ def set_version(self, index, version):
         version_data = version.get("data", dict())
 
         # Compute frame ranges (if data is present)
-        start = version_data.get("startFrame", None)
-        end = version_data.get("endFrame", None)
-        handles = version_data.get("handles", None)
-        if start is not None and end is not None:
+        frame_start = version_data.get("frameStart", None)
```

Comment:
This fails to fall back to the original `startFrame` and thus is not backwards compatible.

This should be:
```python
version_data.get("frameStart", 
                 # backwards compatibility
                 version_data.get("startFrame", None))
```

The same goes for `frameEnd` below.

Reply:
true. well spotted.